client/src/api/planters.js

```diff
@@ -80,4 +59,18 @@ export default {
       },
     }).then(handleResponse).catch(handleError);
   },
+
+  getPlanterRegistrations(planterId) {
+    const registrationQuery =
+      `${process.env.REACT_APP_API_ROOT}/api/planter_registration?filter[where][planterId]=${planterId}`;
+    return fetch(registrationQuery, {
+      method: "GET",
+      headers: { 
+        "content-type": "application/json" ,
```

Comment:
nit, remove quotes since Authorization doesn't have them, or add quotes to it

Reply:
I'll leave this one as it's done this way across the codebase – only using quotes in object keys where necessary (in this case, due to the hyphen).
It would benefit from a tidy up using `prettier`, though.

Reply:
ahh right, my bad, can ignore then